lib/org/chromium/webidl/LocalGitRunner.js

```diff
@@ -5,18 +5,14 @@
 
 foam.CLASS({
   package: 'org.chromium.webidl',
-  name: 'LocalGitIDLFileDAO',
-  extends: 'foam.dao.ProxyDAO',
+  name: 'LocalGitRunner',
+  extends: 'foam.box.Runnable',
 
-  documentation: 'DAO of IDL files backed by a local git repository.',
+  documentation: 'Runnable Box of IDL files backed by a local git repository.',
```

Comment:
nit: This would be a 'box runnable', not a 'runnable box' -- implements not `Box`, but `Runnable` in the box package. But really, it doesn't. See comment above.